lib/tests/unit/connector/connector_test.cc

```diff
@@ -89,17 +109,21 @@ TEST_CASE("Connector::connect", "[connector]") {
 
         wait_for([&c](){return c.isAssociated();});
         auto ws_i = c.getConnectionTimings().getWebSocketInterval();
+        auto ass_timings = c.getAssociationTimings();
+        auto ass_i = ass_timings.getAssociationInterval();
 
         REQUIRE(c.isAssociated());
-
-        REQUIRE(boost::chrono::high_resolution_clock::duration::zero() < ws_i);
+        REQUIRE(ass_timings.completed);
+        REQUIRE(ass_timings.success);
+        REQUIRE(AssociationTimings::Duration_ms::zero() < ass_i);
```

Comment:
This fails on AppVeyor.
I guess the test should not assume that it would take less than 1 ms.
Will use the time_points instead.